packages/core/src/__tests__/__e2e__/dialog/Dialog.cy.tsx

```diff
@@ -184,4 +184,13 @@ describe("GIVEN a Dialog", () => {
       cy.findByRole("button", { name: "Next" }).should("be.focused");
     });
   });
+  describe("WHEN overflowing content is detected", () => {
+    it("THEN it should add padding to the right of the scroll bar", () => {
```

Comment:
Is this better shown as a visual snapshot?

Reply:
the snapshot surfaced the change but the scrollbar doesn't show so it is not super evident, but i can remove this ones if you think they are redundant

Reply:
Happy for this to still be here then